src/components/AcceptTermsForm/index.tsx

```diff
@@ -43,7 +42,6 @@ const AcceptTermsForm = () => {
       </p>
 
       <Checkbox
-        className={styles.checkbox}
```

Comment:
Why doesn't it need a `className` anymore?

Reply:
The style rule added a margin top to the element but then I replaced this specific case and made a generic rule for all checkboxes.